users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -50,18 +50,14 @@ def add_specaug_layer(
         from_list = ["data"]
 
     default_config = {
-        "max_time_num": 3,
-        "max_time": 10,
-        "max_feature_num": 4,
-        "max_feature": 5,
-        "enable_sorting": True,
+        "max_time_num": 1,
+        "max_time": 15,
+        "max_feature_num": 5,
+        "max_feature": 4,
```

Comment:
Is this consistent with the values we used previously?

Reply:
Yes, these are the default values set for specaug_old in `fullsum_ctc_raw_samples.py`